api/admin/get_settings.js

```diff
@@ -1,15 +1,15 @@
 /* eslint-disable no-underscore-dangle */
 
-import { getSettings } from '../../lib/storage';
-import { allLocales as locales } from '../../lib/locale';
+import { getSettings, getLocales } from '../../lib/storage';
 
 module.exports = () => ({
   method: 'GET',
   config: {
     auth: 'jwt'
   },
   path: '/admin/settings',
-  handler: (req, reply) => {
+  handler: async (req, reply) => {
+    const locales = await getLocales();
```

Comment:
Do you have any idea what this compiles down to? I want to be sure it runs on node 4.4

Reply:
I'll test it on node 4.4. Compiles to this:
```js
return {
            method: "GET",
            config: {
                auth: "jwt"
            },
            path: "/admin/settings",
            handler: function() {
                function e(e, n) {
                    return t.apply(this, arguments)
                }
                var t = (0, f.default)(i.default.mark(function e(t, n) {
                    var r, o;
                    return i.default.wrap(function(e) {
                        for (;;) switch (e.prev = e.next) {
                            case 0:
                                return e.next = 2, (0, d.getLocales)();
                            case 2:
                                r = e.sent, o = (0, c.default)(r).map(function(e) {
                                    return {
                                        code: e,
                                        name: r[e]._name
                                    }
                                }), (0, d.getSettings)().then(function(e) {
                                    n((0, u.default)({}, e, {
                                        availableLocales: o
                                    }))
                                });
                            case 5:
                            case "end":
                                return e.stop()
                        }
                    }, e, void 0)
                }));
                return e
            }()
        }
```

Reply:
Hmmm alright that should be good enough 👍  Thanks!